pyre/bench/synth/seqiter_tuple_error_parity.py

```
@@ -0,0 +1,169 @@
+# bench only asserts behaviour where 3.14 and PyPy AGREE; it guards the
+# tuple-subclass override (P1), the live `__length_hint__` (P2) and the
+# type-slot `__getitem__` iteration check (3b).  The 3.14-specific seq-iter
+# `next` behaviour (StopIteration treated as exhaustion, retryable after a
+# non-IndexError) diverges from PyPy and is documented at baseobjspace.rs
+# `next` / `iter_iternext`, so it cannot be asserted against the PyPy oracle
+# here.
+import operator
+
+
+class LenSeq:
+    def __len__(self):
```

Ruff (0.15.18)

[warning] Missing return type annotation for special method `__len__`

Add return type annotation: `int`

(ANN204)